test/e2e-networking-library.sh

```diff
@@ -129,8 +138,13 @@ function setup_ingress_env_vars() {
     export GATEWAY_NAMESPACE_OVERRIDE=contour-external
   fi
   if is_ingress_class gateway-api; then
-    export GATEWAY_OVERRIDE=istio-ingressgateway
-    export GATEWAY_NAMESPACE_OVERRIDE=istio-system
+    if [[ "${GATEWAY_API_IMPLEMENTATION}" == "contour" ]]; then
+      export GATEWAY_OVERRIDE=${GATEWAY_OVERRIDE:-knative-gateway}
+      export GATEWAY_NAMESPACE_OVERRIDE=${GATEWAY_NAMESPACE_OVERRIDE:-contour-external}
+    else
+      export GATEWAY_OVERRIDE=${GATEWAY_OVERRIDE:-istio-ingressgateway}
+      export GATEWAY_NAMESPACE_OVERRIDE=${GATEWAY_NAMESPACE_OVERRIDE:-istio-system}
```

Comment:
I think I'd prefer this function to override the existing values - vs. preserving them